lib/src/ruby/module.cc

```diff
@@ -155,8 +149,7 @@ namespace facter { namespace ruby {
         ruby.rb_gc_register_address(&_on_message_block);
 
         // Install a logging message handler
-        on_message([this](log_level level, string const& message) {
-            auto const& ruby = *api::instance();
```

Comment:
@peterhuene why didn't we inherit ruby from the parent scope? Looks safe enough to me.